### PR TITLE
Add wrapper function for tribe_events_get_the_excerpt

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Barf out, gag me with a spoon! esc_js()!This is for inline javascript, which is against our standards. Use json_encode instead. | [Abstract.php:93](https://github.com/moderntribe/tribe-common/blob/b6e3cd3a057621dc2a8c7792e8fd6c3a1e2e2935/src/Tribe/JSON_LD/Abstract.php#L93)
:x: | There must be a single space after the first parenthesis | [utils.php:430](https://github.com/moderntribe/tribe-common/blob/b6e3cd3a057621dc2a8c7792e8fd6c3a1e2e2935/src/functions/utils.php#L430)

> via jenkins/codesniffer-phpcs

File: src/functions/utils.php [hunk 1]
```diff
@@ -426,3 +426,37 @@ function tribe_post_exists( $post_id_or_name, $post_type = null ) {
 		return ! empty( $found ) ? (int) $found : false;
 	}
 }
+
+if ( ! function_exists('tribe_post_excerpt' ) ) {
```
Comment (bordoni) [inline src/functions/utils.php]: Spacing and we can merge.

**Comment by mitogh** [comment]
Updated!

**Comment by mitogh** [comment]
@bordoni would you mind taking another look over here.

Thanks.

**Pull request merged**